test-support/helpers/ember-power-select.js

```diff
@@ -114,4 +114,24 @@ export default function() {
     }
 
   });
+
+  Ember.Test.registerAsyncHelper('removeSelectedMulOption', function(app, cssPath, value) {
```

Comment:
I'd probably call it `removeMultipleOption`. I find the "selected" redundant because it has to be selected to be removed.